package.json

```diff
@@ -63,6 +63,7 @@
     "test:unit:coverage": "vitest run --coverage",
     "test:unit:update-snapshots": "vitest --update",
     "test-ci": "run-s test:*",
+    "testu": "npm run build:dist; VITEST=1 npm run build:test-dist ; npm run test:unit -- -u",
```

Comment:
Tweak the above line? "test:unit:update-snapshots": "vitest --update"

Reply:
I don't think we should update snapshots in CI. If something changes we should be aware of it. So I'm not in favor of this one.

Reply:
the above line is `ci`, where I don't think we want to do this... but you made another comment too :p